src/util/dump.ts

```diff
@@ -90,7 +90,7 @@ export async function downloadModule(module: string, type: 'js' | 'css') {
     )
   }
 
-  const langMw = MediaWiki.metaData?.langMw || 'en'
+  const langMw = MediaWiki.metaData?.langVar || MediaWiki.metaData?.langMw || 'en'
```

Comment:
Shouldn't we rename the const `lang` or `moduleLang` since it might contain is the variant or the language or the MW?